package.json

```diff
@@ -122,6 +150,51 @@
                 "mac": "Alt+cmd+o",
                 "when": "editorTextFocus"
             }
-        ]
+        ],
+        "menus": {
+            "editor/context": [
+                {
+                    "command": "clangd.typeHierarchy",
+                    "when": "resourceLangId == cpp",
+                    "group": "0_navigation@4"
+                }
+            ],
+            "view/title": [
+                {
+                    "command": "clangd.typeHierarchy.viewParents",
+                    "when": "view == clangd.typeHierarchyView",
+                    "group": "navigation"
+                },
+                {
+                    "command": "clangd.typeHierarchy.viewChildren",
+                    "when": "view == clangd.typeHierarchyView",
+                    "group": "navigation"
+                },
+                {
+                  "command": "clangd.typeHierarchy.close",
+                  "when": "view == clangd.typeHierarchyView",
+                  "group": "navigation"
+                }
+            ],
+            "commandPalette": [
+                {
+                    "command": "clangd.typeHierarchy.viewParents",
+                    "when": "false"
```

Comment:
FWIW call hierarchy exposes the equivalents in the command palette.

It calls these "Calls: Show call hierarchy"/"Calls: Show incoming calls" etc.
We could do something similar with a "Types" prefix...